edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/IDependencyManager.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Hub.Service
+{
+    using Autofac;
+
+    public interface IDependencyManager
+    {
+        void RegisterTo(ContainerBuilder builder);
```

Comment:
Nit: Suggest renaming to Register (fine if you leave it like this)

Reply:
Done